src/rgw/rgw_bucket.cc

```diff
@@ -331,7 +333,7 @@ int rgw_remove_object(rgw::sal::RGWRadosStore *store, const RGWBucketInfo& bucke
 
   rgw_obj obj(bucket, key);
 
-  return store->getRados()->delete_obj(rctx, bucket_info, obj, bucket_info.versioning_status());
+  return store->getRados()->delete_obj(rctx, bucket_info, obj, bucket_info.versioning_status(), 0, ceph::real_time(), nullptr, span_1);
```

Comment:
what are the new parameters on top of `span_1``?

Reply:
these are default ones as c++ doesnt allow named parameter so I had pass the default parameter of the all the optional parameter before actually passing the original span parameter